src/chain/chain.rs

```diff
@@ -2015,82 +2009,4 @@ mod tests {
             check_infos_for_duplication(&chain);
         }
     }
-
-    #[test]
```

Comment:
These removed tests are no longer necessary, right?

Reply:
Yes, they were tested the relocate being signed by Parsec, but we no longer do that (since it is signed when sending the message).